pkg/local_instance.go

```diff
@@ -370,6 +370,11 @@ func (li LocalInstance) Start() error {
 	if err := li.CheckPortsOpen(); err != nil {
 		return err
 	}
+	defer func() { pathx.DeleteIfExists(li.passwordFile()) }()
```

Comment:
pwd file is read by AEM only when it is not initialized / it is ignored on next runs / read only at first run